src/Poly.jl

```diff
@@ -239,6 +239,14 @@ end
 
 isunit(a::PolynomialElem) = length(a) == 1 && isunit(coeff(a, 0))
 
+@doc Markdown.doc"""
+    isconstant(a::PolynomialElem)
+
+Return `true` if the given polynomial is constant, i.e. has length less at most
```

Comment:
Typo "less at most".

Also perhaps refer to `degree` (which every mathematician will understand) instead of `length`?

Indeed, I have to ask: what *is* the length of a polynomial? I couldn't find it documented in the manual, only in indirect references (e.g. the docstring for `degree` defines the degree via the length...). Looking at the code, I see what it means, but isn't this an implementation detail that should be hidden?

To add to my confusion, on  https://nemocas.github.io/AbstractAlgebra.jl/dev/mpolynomial/ it lists`length(f::MPolyElem)` in the APIs but without explanation. Looking at the docstring, it says: 
> Return the number of terms of the polynomial

but that of course is incompatible with the definition for univariate polynomials...?

Reply:
(sorry if `length` of an univariate polynomial is documented in the manual -- I honestly tried to find it, but probably my search skills were not good enough. If it is in there, I'd appreciate a pointer where to find it!)

Reply:
It's defined on the interface page. The length function is not just left undefined for a univariate polynomial in the functionality section of the documentation but the length function is entirely undocumented there. It should be so thanks for picking that up.

I'll add a PR for that today if someone didn't already beat me to it.

Reply:
isconstant was multiply defined. I removed the one with the incorrect docstring.